ClassPaulita/GuessTheCardGame.cs

```diff
@@ -1,10 +1,144 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.IO;
 
 namespace ClassPaulita
 {
     public class GuessTheCardGame
     {
-        private string spacing = "\n\t\t\t\t\t\t";
+        private string _fileName = "scoreBoard.txt";
+        private List<KeyValuePair<int, string>> _scoreBoard = new List<KeyValuePair<int, string>>();
+        private Random _random = new Random();
+        private string _spacing = "\n\t\t\t\t\t\t";
+        
+
+        public GuessTheCardGame()
+        {
+
+            if (File.Exists(_fileName))
+            {
+                StreamReader reader = File.OpenText(_fileName);
+
+                while (true)
+                {
+                    string line = reader.ReadLine();
+
+                    if (line == null)
+                    {
+                        break;
+                    }
+                    else
+                    {
+                        string[] scoreAndName = line.Split(' ');
+
+                        if (scoreAndName.Length == 2)
+                        {
+                            bool isItConvertible = int.TryParse(scoreAndName[0], out int result);
+
+                            if (isItConvertible && scoreAndName[1] != null)
+                            {
+                                _scoreBoard.Add(new KeyValuePair<int, string>(result, scoreAndName[1]));
+                            }
+                        }
+                    }
+                }
+
+                reader.Close();
+            }
+
+            else
+            {
+                File.Create(_fileName);
+            }
+
+            if (_scoreBoard.Count < 5)
+            {
+
+                List<string> extraNames = new List<string> { "Blondie", "Oreo", "Mini", "Ginger", "Shawnikua" };
+                List<int> extraScores = new List<int> { 0, 1, 2, 3, 4, 5 };
+
+                int missingLines = _scoreBoard.Count;
+
+                for (int i = 0; i < 5 - missingLines; i++)
+                {
+                    int randomNameIndex = _random.Next(extraNames.Count - 1);
+
+                    int randomScoreIndex = _random.Next(extraScores.Count - 1);
```

Comment:
Yeap. Fixing it...

Reply:
Well, I did it that way in case I want to add more names. That way I do not have change the count.

Reply:
scores don't effect how many names you have, but if you want the score range to be the size of the names just do this:
```
int randomScore = _random.Next(extraNames.count); // 0 - # of names
```

Reply:
You marked this a resolved, but I'm not seeing a fix

Reply:
Also currently you have the random score going all the way up to 51, which is the highest possible score. Meaning, the first time they play this game and it creates the scores, it could start off with a score they could never beat.

Reply:
Ok. I fixed it.